core/common/src/ClipStyle.ts

```diff
@@ -97,6 +98,62 @@ export class CutStyle {
   }
 }
 
+/** Wire format describing a [[ClipHighlight]].
+ * @see [[ClipStyleProps.ClipHighlight]].
+ * @public
+ * @extensions
+ */
+export interface ClipHighlightProps {
+  /** Color to highlight the edge */
+  color?: RgbColorProps; // default white
+  /** Width of highlight in pixels */
```

Comment:
Is this more accurately the distance a pixel can be to be considered to intersect the clip plane and therefore get highlighted instead of clipped?